subsys/bluetooth/host/conn_internal.h

```diff
@@ -115,10 +116,10 @@ struct bt_conn {
 	u16_t		        rx_len;
 	struct net_buf		*rx;
 
-	/* Sent but not acknowledged TX packets */
+	/* Sent but not acknowledged TX packets with a callback */
 	sys_slist_t		tx_pending;
-	/* Acknowledged but not yet notified TX packets */
-	struct k_fifo		tx_notify;
+	/* Sent but not acknowledged TX packets without a callback */
+	u32_t                   pending_no_cb;
```

Comment:
Could we extend this comment to say that this is before the tx_pending?

i.e /* Sent but not acknowledged TX packets without a callback before the first element  with a callback */ ?

Reply:
sure, that does indeed make it clearer